js/ui/codingManager.js

```diff
@@ -692,41 +984,36 @@ const CodingManager = new Lang.Class({
         return false;
     },
 
-    rotateInCompleted: function(actor) {
-        if (this._removeEffect(this._rotateInActors, actor)) {
-            Tweener.removeTweens(actor);
-            actor.opacity = 255;
-            actor.rotation_angle_y = 0;
-            actor.set_cull_back_face(false);
-        }
+    removeBuilderWindow: function(actor) {
+        let window = actor.meta_window;
+        let isSpeedwagonForBuilder = _isBuilderSpeedwagon(window);
+
+        if (!(_isBuilder(window.get_flatpak_id()) ||
+              isSpeedwagonForBuilder))
```

Comment:
This check is done differently (as an AND of negations) above. Apply De Morgan's law to either :-)